package.json

```diff
@@ -35,7 +35,6 @@
     "jest": "^22.4.4",
     "jquery": "^3.1.1",
     "laravel-mix": "2.1.*",
-    "lodash": "^4.17.10",
```

Comment:
Since you moved `lodash` over to `dependencies`, I think you may as well move `jquery`, `moment`, `moment-timezone` and `vue` over as well. Though for this repo it doesn't really matter where it goes.

Reply:
Hmm, I think I moved lodash over as we have it as a main dep modeler while it was in dev here. I can look at moving the others over later if need be...

Reply:
I want to try to update the deps in this repo so maybe I can come back to this then.